pyproject.toml

```
@@ -228,19 +229,24 @@ ignore = [
   "ICN",      # flake8-import-conventions
   "PYI",      # flake8-pyi
   "NPY002",   # Replace legacy `np.random.random` call with `np.random.Generator`
+  "W291",     # Trailing white space
 ]
 exclude=['doc/conf.py', '__init__.py']
 
 # Need to select D above
 #[tool.ruff.lint.pydocstyle]
 #convention = "numpy"
 
+[tool.ruff.lint.pycodestyle]
+max-doc-length = 110
```

Should we set the doc line length to 88 to match the code?
The previous setting was 110 (which I think you said was the default also).
Seems to me it'd be nice to have it the same for docs and code, but maybe I'm missing something.
Are there lots more changes this way?
```suggestion
max-doc-length = 88
```

---

oof... 88-char limit leads to 307 changes needed.
Although most of them would make it read better, that's a lot of churn.
(Some even have grammar mistakes that fixing would make within the length limit.) :)

I'm fine with leaving this at 110 -- and I'm willing to do the fixes too (or help do or have someone else do).
:)